test/files.go

```diff
@@ -68,3 +71,24 @@ func readFileContents(fileName string) (contents []byte, err error) {
 	}
 	return contents, err
 }
+
+func describeYamlError(yamlError error) string {
+	matched, err := regexp.MatchString(`.*int was used where sequence is expected.*`, yamlError.Error())
+	if err != nil {
+		return err.Error()
+	}
+	if matched {
+		return "This might refer to a \"status\" line being '200', where it should be '[200]'. " +
+			"The default \"status\" is a list now."
+	}
+	matched, err = regexp.MatchString(`.*cannot unmarshal \[]interface {} into Go struct field FTWTest.Tests of type string.*`, yamlError.Error())
+	if err != nil {
+		return err.Error()
+	}
+	if matched {
+		return "This might refer to \"data\" on the test being a list of strings instead of a proper YAML multiline. " +
+			"To fix this, convert this \"data\" string list to a multiline YAML and this will be fixed. "
```

Comment:
How would one do that? With `|`? Add a short example.

Reply:
Added!